security-release-process.md

```diff
@@ -161,7 +161,7 @@ Note: The kubernetes-security GitHub org is co-owned and viewable by the SRC and
 These steps should be completed within the 1-7 days of Disclosure.
 
 - The Fix Lead and the Fix Team will create a [CVSS](https://www.first.org/cvss/specification-document) score using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). They will also use the [Severity Thresholds - How We Do Vulnerability Scoring](#severity-thresholds---how-we-do-vulnerability-scoring) to determine the effect and severity of the bug. The Fix Lead makes the final call on the calculated risk; it is better to move quickly than make the perfect assessment.
-- The Fix Lead will request a CVE from the [Kubernetes CVE Numbering Authority](cve-requests.md).
+- The Fix Lead will [Assign a CVE ID to vulnerability from the CVE Numbering Authority](/cna-handbook.md#assign-a-cve-id-to-vulnerability).
```

Comment:
```suggestion
- The Fix Lead will [Assign a CVE ID to the vulnerability from the CVE Numbering Authority](/cna-handbook.md#assign-a-cve-id-to-vulnerability).
```
just a nit, think we need to update the title of the linked section too. But we can do that in another PR if you'd like.

Reply:
done. PTAL

Reply:
LGTM! Thanks!